### DISCUSSION TITLE
`quick_access_cards`, `extra_cards`, chips should be now under `homeView` option

### DISCUSSION THREAD

**Discussion by AalianKhan** [DigiLive/mushroom-strategy#86] [Ideas]
Hello,

Since now we have an `homeView` option, I think the following options should fall under the homeView option because they are all editing the Home View

- `chips`
- `quick_access_Cards` (Maybe rename it to header cards?)
- `extra_cards` (Mayber rename it to footer cards?)

What do you guys think?

**Comment by DigiLive** [comment]
It makes sense to me.
However, I would keep the current format also valid for a while with a deprecation notice in the console if they still use it.

**Reply by dipseth** [reply]
are these homeview options available when using the v1.0.1 release.

I tried but they seemed to have no affect.

**Reply by DigiLive** [reply]
No. They haven't been released yet, but are available in the main branch.
See issue #82 for more information.
